src/QueryBuilder.php

```diff
@@ -184,10 +180,8 @@ private function buildCustomConditions(QueryBlock $queryBlock): string
     {
         $conditions = [];
 
-        $field_slug = tao_helpers_Slug::create($queryBlock->getField());
-
         foreach (self::CUSTOM_FIELDS as $customField) {
-            $conditions[] = sprintf('%s_%s:"%s"', $customField, $field_slug, $queryBlock->getTerm());
+            $conditions[] = sprintf('%s_%s:"%s"', $customField, $queryBlock->getField(), $queryBlock->getTerm());
```

Comment:
I imagine yes, but just to be 100% sure, will the `$queryBlock->getField()` be always URI encoded from tao?